src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -18,21 +15,23 @@ public List<TaskDto> getTasks() {
         return new ArrayList<>();
     }
 
-
+    @GetMapping
```

Comment:
są teraz 2 get mappingi z takim samym adresem... raczej to nie zadziała prawidłowo. @GetMapping dla konkretnego ID powinien zawierać albo RequestParam albo PathVariable

Reply:
Tak, wyjaśnili to po fakcie w submodule 8.5 :)
Poprawione w nowym commicie. Proszę o rewizję.